presto-main/src/test/java/com/facebook/presto/type/TestRowOperators.java

```diff
@@ -503,7 +593,10 @@ public void testRowCast()
         assertFunction("cast(row(1,null,3) as row(aa bigint, bb boolean, cc boolean)).aa", BIGINT, 1L);
         assertFunction("cast(row(null,null,null) as row(aa bigint, bb boolean, cc boolean)).aa", BIGINT, null);
 
-        assertInvalidFunction("CAST(ROW(1, 2) AS ROW(a BIGINT, A DOUBLE)).a");
```

Comment:
I'll look at the whole PR when I have a chance, but since you've deleted this, you can now rid of the deprecated assertInvalidFunction that this is using from AbstractTestFunctions and FunctionAssertions.  This was the only place using it.

Reply:
Ok, thank you for the reminder, I will remove them.